core/db/migrate/20130807024301_upgrade_adjustments.rb

```diff
@@ -14,7 +14,8 @@ def up
 
     # Tax adjustments have their sources altered
     Spree::Adjustment.where(:originator_type => "Spree::TaxRate").find_each do |adjustment|
-      adjustment.source = adjustment.originator
+      adjustment.source_id = adjustment.originator_id
+      adjustment.source_type = "Spree::TaxRate"
       adjustment.save
```

Comment:
not that this was written by you, but i'm curious why we don't care about whether it was able to save here. maybe `save!` would be better?